java-checks-test-sources/src/main/java/checks/TestsInProductionFile.java

```diff
@@ -0,0 +1,10 @@
+package checks;
+
+import org.junit.jupiter.api.Test;
+
+public class TestsInProductionFile { // Noncompliant [[sc=14;ec=35]] {{Move this test class to a separate assembly in this solution.}}
```

Comment:
The issue message seems to come from C# world: `Move this test class to a separate assembly in this solution.`. I think we should have a different one for java, as this vocabulary does not apply to our context.